src/main/java/org/sqlite/core/NativeDB.java

```diff
@@ -36,20 +36,8 @@ public final class NativeDB extends DB {
     /** SQLite connection handle. */
     private long pointer = 0;
 
-    private static boolean isLoaded;
-    private static boolean loadSucceeded;
-
-    static {
-        if ("The Android Project".equals(System.getProperty("java.vm.vendor"))) {
-            System.loadLibrary("sqlitejdbc");
-            isLoaded = true;
-            loadSucceeded = true;
-        } else {
-            // continue with non Android execution path
-            isLoaded = false;
-            loadSucceeded = false;
```

Comment:
i don't see where that bit ended up being moved to?

Reply:
it is very much relevant, without this bit the library won't run anymore on Android.